config/environment.js

```diff
@@ -181,6 +181,7 @@ module.exports = function(environment) {
                 dashboard: 'ember_home_page',
                 home: 'ember_home_page',
                 institutions: 'ember_institutional_landing_page',
```

Comment:
I don't think we have a flag for the route, only the updated flag for the menu item which will _probably_ be `institutions_nav_bar`. https://github.com/CenterForOpenScience/osf.io/pull/8333/files#diff-758f58f82207e3674b3105d2b3d4ce7aR127

Reply:
Indeed. I wonder if we *should* flag the route though. In this case, the Ember app wouldn't reload, but just go to page not found.

Reply:
The thought was that, because the page doesn't exist, if we don't tell anyone about it, there's no reason to give it a flag. It will either need to be on for everyone or no-one, but not showing the menu item is a valid thing to hide to make it easier to sync between teams.